paasta_tools/vitesscell_tools.py

```diff
@@ -115,6 +126,7 @@ def get_cell_config(
             },
             extraLabels=labels,
             replicas=replicas,
+            scale_selector=",".join([f"{k}={v}" for k, v in labels.items()]),
```

Comment:
I could replace this with just service and instance labels, e.g. something like this:
```suggestion
            scale_selector=",".join([f"{k}={v}" for k, v in {
                paasta_prefixed("service"): self.service,
                paasta_prefixed("instance"): self.instance,
            }.items()]),
```
Let me know what you think.

Reply:
I'm leaving it like this for now. I think it is safe. We can change it later if necessary.